gratipay/billing/exchanges.py

```diff
@@ -74,6 +74,7 @@ def create_card_hold(db, participant, amount):
 
     hold = None
     error = ""
+    ref = ""
```

Comment:
I needed to set this (but it could be Null) because it kept giving the error that the variable was being referenced before being assigned